shared/subprocess/manager.go

```diff
@@ -4,19 +4,45 @@
 package subprocess
 
 import (
+	"io"
 	"io/ioutil"
+	"os"
 
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v2"
 )
 
 // NewProcess is a constructor for a process object. Represents a process with argument config. Returns an address to process
 func NewProcess(name string, args []string, stdoutPath string, stderrPath string) (*Process, error) {
```

Comment:
Whilst we're doing this it would be good to update the description of this function indicating that stdoutPath and stderrPath are optional.

Reply:
Fixed.